configure.ac

```diff
@@ -274,6 +274,14 @@ AC_ARG_ENABLE([warnings-as-errors],
                                ])],
               [WARNINGS_AS_ERRORS=$enableval])
 
+## ----------------------------------------------------------------------
+## Check if we should build the HDF5 API tests
+AC_ARG_ENABLE([api-tests],
```

Comment:
This section should likely be moved down below the additional processing of WARNINGS_AS_ERRORS below.